lib/zoom/client.rb

```diff
@@ -29,20 +29,32 @@ class Client
     def headers
       {
         'Accept' => 'application/json',
-        'Content-Type' => 'application/json',
+        'Content-Type' => 'application/json'
```

Comment:
I would leave this comma to keep the diff churn down. As an aside, I am in favor of trailing commas to keep diff churn down when adding another element to these types of collections.